### PR TITLE
Upgrade to latest deps: prevent false positives on modern Coffee syntax

### PR DESCRIPTION
**Pull request by elektronik2k5**
Such as import/export

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "linter-coffeelint",
   "main": "./lib/init",
   "activationCommands": {},
-  "version": "1.1.2",
+  "version": "1.2.0",
```
Comment (Arcanemagus) [inline package.json]: Don't update the version, this will be done when releasing a new version.

File: package.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
     }
   },
   "dependencies": {
-    "atom-package-deps": "3.0.6",
-    "coffeelint": "1.9.7",
-    "resolve": "1.1.5"
+    "atom-package-deps": "4.3.1",
```
Comment (Arcanemagus) [inline package.json]: Can this be made a range please.

File: package.json [hunk 3]
```diff
@@ -20,8 +20,8 @@
...
+    "coffeelint": "1.16.0",
+    "resolve": "1.1.7"
```
Comment (Arcanemagus) [inline package.json]: Again, this should be a range as it's a utility library.

**Comment by Arcanemagus** [comment]
Very interesting... it looks like somebody disabled GreenKeeper on this repository, so none of the PRs that should have been coming in updating this have well... been coming in. 😠

**Comment by Arcanemagus** [comment]
Went ahead and made the changes in #70, thanks for bringing this up!

**Comment by Arcanemagus** [comment]
Published v1.2.0, now to add specs and clean up the rest of this mess.

**Comment by elektronik2k5** [comment]
@Arcanemagus awesome, thanks!

**Pull request closed without merging**